app/main/posts/views/card.html

```diff
@@ -3,7 +3,7 @@
     <div class="listing-item-select">
         <input ng-show="canSelect" type="checkbox" checklist-value="post.id" checklist-model="selectedPosts" ng-click="stopClickPropagation($event)">
     </div>
-    <div class="postcard-body" ng-click="clickAction($event)">
+    <div class="postcard-body" tabindex="0" role="button" ng-click="clickAction($event)">
```

Comment:
@Angamanga  @rowasc, Please can I change this to button tag as it's not clickable(using space-tab or enter) for accessibility, I tried changing it but the styles are not rendering well, I need your permission to override it 

## View with div element
![button-cc](https://user-images.githubusercontent.com/66112760/97093743-8d5aec00-1646-11eb-908c-8ad7b1d16f35.gif)

## View with button element

![button-pc](https://user-images.githubusercontent.com/66112760/97093755-ab285100-1646-11eb-8c20-b44a37f7619b.gif)

With just space bar and enter key I was able to tab through the post on button element.

Reply:
@bimbolabuari Changing all the cards to buttons is quite a big change and requires some speccing before we go on that route. It is something I am thinking dealing with during the internship. For now, you could check the examples here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Examples to figure out how to make the div "clickable" with the space and enter-keys 🙏

Reply:
@Angamanga, I will do this ASAP, thanks for the feedback!